misc-tools/dj_make_chroot.in

```diff
@@ -149,6 +149,14 @@ fi
 [ -z "$CHROOTDIR" ] && error "No chroot directory given or default known."
 [ -z "$ARCH" ]      && error "No architecture given or detected."
 
+ARCHLIST=$DEBIAN_ARCHLIST
+if [ "$DISTRO" = "Ubuntu" ]; then
+    ARCHLIST=$UBUNTU_ARCHLIST
+fi
+if [ -n "${ARCHLIST##*"$ARCH"*}" ]; then
```

Comment:
Nitpick, but this would not match for `arm` (since it's a substring e.g. of `armhf`) which is not a valid architecture.

Reply:
Are there actually machines still with an arm processor (instead of arm64?)

But I'll see if there is a nice clean (readable) solution for this.